src/spurt/mcf/utils.py

```diff
@@ -62,16 +68,29 @@ def flood_fill(indata: np.ndarray, links: np.ndarray, flows: np.ndarray):
         errmsg = f"Dimension mismatch - {links.shape} vs {flows.shape}"
         raise ValueError(errmsg)
 
+    if len(indata) == len(links):
```

Comment:
This is a highly unlikely edge case. Any suggestions for how to handle this? Add another kwarg that by default is set to "auto" or else explicitly set to "points" or "links"?

Reply:
Yeah that sounds like a good solution to me. I think we should prefer to avoid using the "auto" mode, though, in case this issue ever comes up in practice.

Reply:
Have added this to `feat/dirichlet`